plugins/contactpressuresensorsarray/include/gazebo/ContactPressureSensorsArray.hh

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2018 Fondazione Istituto Italiano di Tecnologia RBCS & iCub Facility & ADVR
+ * Authors: see AUTHORS file.
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ */
+
+#ifndef GAZEBOYARP_CONTACTPRESSURESENSORSARRAY_HH
+#define GAZEBOYARP_CONTACTPRESSURESENSORSARRAY_HH
+
+#include <gazebo/common/Plugin.hh>
+#include <string>
+#include <yarp/dev/PolyDriverList.h>
+
+#include <sdf/sdf_config.h>
+#include <sdf/Element.hh>
+#include <sdf/Param.hh>
+
+
+namespace sdf 
+{
+    class Element;
+}
```

Comment:
If you include the `sdf/Element.hh` header, there is no need to forward declare the `sdf::Element` class.